hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/ControllerManagement.java

```diff
@@ -93,17 +93,17 @@
             @NotNull Collection<Long> moduleId);
 
     /**
-     * Simple addition of a new {@link ActionStatus} entry to the {@link Action} .
-     * No state changes.
+     * Simple addition of a new {@link ActionStatus} entry to the {@link Action}
+     * . No state changes.
```

Comment:
line break probably caused by some unintended whitespace

Reply:
formatted it correctly